src/main/js/store/actions/globalConfiguration.js

```diff
@@ -186,18 +191,24 @@ export function updateConfig(config) {
         }
         request.then((response) => {
             if (response.ok) {
-                response.json().then((data) => {
-                    const newId = data.id;
-                    const updatedConfig = FieldModelUtilities.updateFieldModelSingleValue(config, 'id', newId);
-                    dispatch(configUpdated(updatedConfig));
-                    return newId;
-                }).then((data) => {
-                    dispatch(refreshConfig(data));
-                });
+                response.json()
+                    .then((data) => {
+                        const newId = data.id;
+                        const updatedConfig = FieldModelUtilities.updateFieldModelSingleValue(config, 'id', newId);
+                        dispatch(configUpdated(updatedConfig));
+                        return newId;
+                    })
+                    .then((data) => {
+                        dispatch(refreshConfig(data));
+                    });
             } else {
+                if (id) {
+                    dispatch(refreshConfig(id));
+                }
                 handleFailureResponse(dispatch, response);
             }
-        }).catch(console.error);
+        })
+            .catch(console.error);
```

Comment:
Is there something wrong with your formatter? It looks like you updated a lot of JS files and the only change was this strange formatting